plugin/src/test/kotlin/com/exactpro/th2/gradle/BaseTh2PluginTest.kt

```diff
@@ -86,4 +84,32 @@ internal class BaseTh2PluginTest {
             subProject.pluginManager.apply(BaseTh2Plugin::class.java)
         }
     }
+
+    private fun <T : Plugin<*>> extractedPlugin(
```

Comment:
note: `extractedPlugin` does not say anything. Maybe we should rename it to `assertHasPlugin`